src/Symfony/Component/HttpKernel/HttpCache/Store.php

```diff
@@ -106,7 +106,10 @@ public function unlock(Request $request)
 
     public function isLocked(Request $request)
     {
-        return is_file($this->getPath($this->getCacheKey($request).'.lck'));
+        $path = $this->getPath($this->getCacheKey($request).'.lck');
+        clearstatcache(true, $path);
+
+        return is_file($path);
```

Comment:
Is the file check absolutely necessary or couldn't we just call `file_exists()` which doesn't use the stat cache?